src/HostManager.cpp

```diff
@@ -131,15 +132,14 @@ void HostManager::changeAllHostColour(const Host* pHost)
         return;
     }
     //change all main and subconsoles color
-    const QList<QSharedPointer<Host>> hostList = mHostPool.values();
-    for (int i = 0; i < hostList.size(); i++) {
-        hostList.at(i)->mpConsole->changeColors();
+    for (QSharedPointer<Host> iHost : mHostPool.values()) {
```

Comment:
What does iHost mean?

Reply:
Not sure, where I got that from. 
Changing to host seems better.